SPECS/xapi.spec

```diff
@@ -1375,6 +1404,268 @@ Coverage files from unit tests
 %{?_cov_results_package}
 
 %changelog
+* Tue Apr 15 2025 Gaëtan Lehmann <gaetan.lehmann@vates.tech> - 25.6.0-1.1
+- Upstream changelog:
+  * Mon Feb 10 2025 Vincent Liu <shuntian.liu2@cloud.com> - 25.6.0-1
+  - CP-52114: Add pool.license_server for pool level licensing
+  - CP-52116: Support pool level licensing data in Host.apply_edition
+  - CP-51209: add hooks lock_acquired/released for bpftrace
+  - Hugo docs update
+  - CA-405628: unmount/detach PVS cache VDI before destroying
+  - xenopsd tests: split suite into 3 executables
+  - CP-53335, topology: do not raise exception when loading invalid distance matrices
+  - test_topology: reorganise test cases
+  - Fix CI: Re-enable running shellcheck even when only docs changed
+  - CP-49141: Mark the DB lock as high priority: try to avoid voluntary Thread.yield while we hold it
+  - CP-49140: prepare for database optimizations
+  - Document Xapi's auto-generated modules
+  - CA-405754: Update xapi-storage-script state.db
+  - Add Pool_role.is_master benchmarks
+  - Optimize fastpath in Pool_role
+  - CA-405971: avoid calling DB functions when in emergency mode
+
```

Comment:
We remove empty lines when we quote the upstream changelog

Reply:
Ah! That's what I did first, then noticed that Yann kept them in the last update, so I reworked the changelog to keep them :-)
I'm fixing that